pokemongo_bot/__init__.py

```diff
@@ -732,6 +732,8 @@ def _setup_api(self):
         # chain subrequests (methods) into one RPC call
 
         self._print_character_info()
+        if self.config.pokemon_bag_show_at_start and self.config.pokemon_bag_pokemon_info:
+            self._print_list_pokemon()
```

Comment:
It is not clear what will happen if "show_at_start" is not set in config. Perhaps it would be best to set it to default false unless True is passed in by the config.